src/containers/AccountConnection.jsx

```diff
@@ -130,16 +130,19 @@ class AccountConnection extends Component {
     return this.store.updateAccount(connector, account, values)
     .then(account => {
       this.setState({ account: account })
-      return this.store.runAccount(connector, account, this.props.disableSuccessTimeout)
```

Comment:
You can't remove this attribute and this is not related to the subject of this PR. I suppose this is a mistake.